packages/wdio-browserstack-service/src/index.ts

```diff
@@ -3,9 +3,14 @@
 import BrowserstackLauncher from './launcher.js'
 import BrowserstackService from './service.js'
 import type { BrowserstackConfig } from './types.js'
+import { configure } from './log4jsAppender.js'
+import logReportingAPI from './logReportingAPI.js'
 
 export default BrowserstackService
 export const launcher = BrowserstackLauncher
+export const log4jsAppender = { configure }
+export const BStackTestOpsLogger = logReportingAPI
```

Comment:
Why are these exported? Are these suppose to be used by users? If so we should provide docs for them right?

Reply:
Yes @christian-bromann these are supposed to be used by user if they want support for `log4js` and `winston-logger`.  Will create a separate PR for docs, as also need to make other doc changes for observability as well